CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOSTests/CBLTestServer_iOSTests.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Should we leave these files as it is or remove them? we can have it as it is and we can add tests in the future or can remove as our client tests are good enough to cover

Reply:
These are autogenerated for the app. Its ok to leave them there. There are no tests in the file.